doc/exporter-guide.md

```diff
@@ -0,0 +1,49 @@
+# Exporter Developer Guide
+
+An exporter sends traces and metrics to any backend that is capable of consuming them. The exporter itself can change without requiring a change in your client code.
+
+OpenTelemetry exporters can be contributed by anyone, and we provide support for several open source backends and vendors out-of-the-box, example Zipkin, Jaeger, Prometheus. This document will provide a means for developers to create their own. We strongly recommended to create a dedicated package for newly added exporter, example: `@opentelemetry/exporter-myexporter`.
+
+## Tracing
+
+`SpanExporter` defines the interface that protocol-specific exporters must implement so that they can be plugged into OpenTelemetry SDK and support sending of spans data. Exporters should expect to receive only sampled-in ended spans. Exporters must not throw. Exporters should not modify spans they receive.
+
+Each exporter must implement `SpanExporter`, which contain `export` and `shutdown` methods:
```

Comment:
If you implement my above suggestion, this sentence should be changed as it becomes redundant.

```suggestion
The current `SpanExporter` interface (`0.2.0`) contains 2 methods:
```